daisy_workflows/image_build/install_package/cos/package_replacement/install_go.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# This script installs go, it's a helper script for compiling the debian pkg.
+
+# Installs a specific version of go for compilation, since availability varies
+# across linux distributions. Needs curl and tar to be installed.
+
+# Takes an argument for go installation path. Recommended: "/tmp/go".
+
+install_go() {
+  local arch="amd64"
+  if [[ `uname -m` == "aarch64" ]]; then
+    arch="arm64"
+  fi
+
+  local GOLANG="go1.21.4.linux-${arch}.tar.gz"
```

Comment:
yet another go toolchain custom install, why did you pick `1.21.4`?

Reply:
https://github.com/GoogleCloudPlatform/guest-test-infra/blob/master/packagebuild/common.sh

We followed the go installation function from the guest-test-infra repo. Will add licensing here as well

Reply:
Done

Reply:
Can you make it consistent with the version we have in that file?